polar-core/src/vm.rs

```diff
@@ -1485,7 +1485,7 @@ impl PolarVirtualMachine {
                             s.chars()
                                 .map(|c| c.to_string())
                                 .map(Value::String)
-                                .filter(|c| !is_ground || term.value() == c)
+                                .filter(|c| !is_ground || c == value)
```

Comment:
To ensure I have this correct: `term.value()` is the entire `in` operation, right? Now we're comparing `c` (a character produced by iterating over the string) and `value`, the LHS of the `in` operation?

Reply:
Yep! The previous behaviour made no sense, but it never got past the `!is_ground` check.